stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MessageReplyView.kt

```diff
@@ -71,9 +69,14 @@ internal class MessageReplyView : FrameLayout {
         }
     }
 
+    /**
+     * @param message [Message] that was replied to.
+     * @param isMine If the message containing the reply was current users or not.
+     * @param style The style to be applied to the view.
+     */
     fun setMessage(message: Message, isMine: Boolean, style: MessageReplyStyle?) {
         setUserAvatar(message)
-        setAvatarPosition(isMine)
+        setAvatarPosition(message.isMine())
```

Comment:
do we need this change?

Reply:
yes, if you look at the description, isMine is the bool of the message containing the replay, where as message is the quoted message. We need to check against the quoted message to show the avatar on the right side.